WoWsShipBuilder.ViewModels/ShipVm/DepthChargeDamageDistributionChartViewModelBase.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using ReactiveUI;
+using WoWsShipBuilder.Core.DataContainers;
+using WoWsShipBuilder.ViewModels.Base;
+
+namespace WoWsShipBuilder.ViewModels.ShipVm;
+
+public class DepthChargeDamageDistributionChartViewModelBase : ViewModelBase
+{
+    public DepthChargeDamageDistributionChartViewModelBase(DepthChargeDataContainer depthCharge)
+    {
+        DcDmg = depthCharge.Damage;
+        SplashRadius = depthCharge.DcSplashRadius;
+        PointsOfDmg = depthCharge.PointsOfDmg;
+    }
+
```

Comment:
Is this really necessary? Can these values ever change? If not, use a simple record class instead of a viewmodel.
Immutable values do not need a viewmodel with property changed notifications.